tools/check-yaml-syntax

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh -e
 # fall back to find if there is no git, e.g. in package builds
-"${1:-yamllint}" --strict $(git ls-files "*.yml" "*.yaml" 2>/dev/null || find -name '*.y*ml')
+"${1:-yamllint}" --strict "$(git ls-files "*.yml" "*.yaml" 2>/dev/null || find . -name '*.y*ml')"
```

Comment:
Is quoting the command substitution the intended behavior?  Looking at the ci error

`No such file or directory: '.github/stale.yml\n.github/workflows/ci.yml\n.github/workflows/ci_extended.yml\n.hadolint.yaml\n.mergify.yml\ncodecov.yml\ndependencies.yaml'`

it seems that the command needs the splitted `$()` output and not the single string `"$()"` one.

Reply:
yes, of course. I fixed that and changed accordingly same as I did for the python style check that was added in the meantime